src/lib.rs

```diff
@@ -1032,3 +1042,69 @@ mod impl_serde {
         }
     }
 }
+
+#[cfg(feature = "zerocopy")]
+mod impl_zerocopy {
+    use super::{BitFlag, BitFlags};
+    use zerocopy::{FromZeros, Immutable, TryFromBytes, Unaligned};
+
+    // All zeros is always valid
+    unsafe impl<T> FromZeros for BitFlags<T>
+    where
+        T: BitFlag,
+        T::Numeric: Immutable,
+        T::Numeric: FromZeros,
+    {
+        fn only_derive_is_allowed_to_implement_this_trait() {}
```

Comment:
I think this could use a comment linking to google/zerocopy#287, so that this doesn't scream "crimes! damn crimes!" at anyone who looks at this later.